installers/olm/Makefile

```diff
@@ -0,0 +1,133 @@
+NAME ?= percona-server-mongodb-operator
+IMAGE_TAG_OWNER ?= perconalab
+IMAGE_TAG_BASE ?= $(IMAGE_TAG_OWNER)/$(NAME)
+SED := $(shell which gsed || which sed)
+VERSION ?= $(shell git rev-parse --abbrev-ref HEAD | $(SED) -e 's^/^-^g; s^[.]^-^g;' | tr '[:upper:]' '[:lower:]')
+IMAGE ?= $(IMAGE_TAG_BASE):$(VERSION)
+MODE ?= namespace
+DEPLOYDIR = ./deploy
+
+BUNDLEDIR = $(DEPLOYDIR)/csv/redhat
+BUNDLE_CHANNELS := --channels=stable
+BUNDLE_DEFAULT_CHANNEL := --default-channel=stable
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.23
+.DEFAULT_GOAL := help
+.SUFFIXES:
+
+CONTAINER ?= docker
+OPENSHIFT_VERSIONS ?= v4.12-v4.15
+PACKAGE_CHANNEL ?= stable
+MIN_KUBE_VERSION ?= 1.24.0
+DOCKER_DEFAULT_PLATFORM ?= linux/amd64
+SHELL := /bin/bash
+REPO_ROOT = $(shell git rev-parse --show-toplevel)
+
+export VERSION
+export BUNDLE_REPO
+export OPENSHIFT_VERSIONS
+export PACKAGE_CHANNEL
+export MIN_KUBE_VERSION
+export DOCKER_DEFAULT_PLATFORM
+export MODE
+
+REPO_ROOT = $(shell git rev-parse --show-toplevel)
+
+distros = community redhat marketplace
+
+check-version:
+ifndef VERSION
+	$(error VERSION is not set)
+endif
+
+KUSTOMIZE = $(REPO_ROOT)/bin/kustomize
+kustomize: ## Download kustomize locally if necessary.
```

Comment:
This does not work for me, when I  execute `make bundles` I get an error in this line:
`/bin/bash: ..repos/percona-server-mongodb-operator/bin/kustomize: No such file or directory`

Yes, I don't have kustomize in bin/ folder, but this `go-get-tool` should be able to download it

Reply:
I see, I will check it too. And I agree that go-get-tool should download it.